psi-ms.obo

```diff
@@ -21880,6 +21880,26 @@ name: Lipid shorthand identification confidence - Complete structure
 def: "Lipid shorthand identification confidence level 'Complete structure'." [PMID:33037133]
 is_a: MS:1003310 ! Lipid shorthand identification confidence level
 
+[Term] 
+id: MS:1003327 
+name: number of identified protein groups 
+def: "The number of protein groups that pass the threshold to be considered identified with sufficient confidence." [PSI:PI]  
+is_a: MS:1002405 ! protein group-level result list attribute 
+is_a: MS:4000003 ! single value 
+relationship: has_metric_category MS:4000008 ! ID based metric 
+relationship: has_value_type xsd:int ! The allowed value-type for this CV term 
+relationship: has_units UO:0000189 ! count unit 
+
+[Term] 
+id: MS:1003328 
+name: count of identified proteoforms 
```

Comment:
**nitpick:** Use "number" instead of "count" here to be consistent?

Reply:
:+1: